queue_services/business-pay/src/business_pay/services/__init__.py

```diff
@@ -0,0 +1,53 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the BSD 3 Clause License, (the "License");
+# you may not use this file except in compliance with the License.
+# The template for the license can be found here
+#    https://opensource.org/license/bsd-3-clause/
+#
+# Redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the
+# following conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice,
+#    this list of conditions and the following disclaimer.
+#
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the following disclaimer in the documentation
+#    and/or other materials provided with the distribution.
+#
+# 3. Neither the name of the copyright holder nor the names of its contributors
+#    may be used to endorse or promote products derived from this software
+#    without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS “AS IS”
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Services used by Business-Pay."""
+from gcp_queue import GcpQueue
+
+from .gcp_auth import verify_gcp_jwt
+from .nats_queue import NatsQueue
+
+gcp_queue = GcpQueue()
+nats_queue = NatsQueue()
+
+
+def create_filing_msg(identifier):
+    """Create the filing payload."""
+    filing_msg = {"filing": {"id": identifier}}
+    return filing_msg
+
+
+def create_email_msg(identifier, filing_type):
+    """Create the email message payload."""
+    email_msg = {"email": {"filingId": identifier, "type": filing_type}}
+    return email_msg
```

Comment:
I think we need to add `option`?  Emailer looks at that.

create_email_msg in queue common looks like this right now

https://github.com/bcgov/lear/blob/d6f8e6ec24e1603d39720f85205a6321dc441be3/queue_services/common/src/entity_queue_common/messages.py#L62-L65

Reply:
I think that's only used by the names processor? 
Names has been moved to their own emailer, so this shouldn't receive those anymore.
I'll look more

Reply:
Think you'll need it even for non-names stuff.  Took a closer look.

![image](https://github.com/bcgov/lear/assets/6685223/e2ea749a-570e-4f31-940e-ac7d595b869e)

![image](https://github.com/bcgov/lear/assets/6685223/8b93395e-fa3f-4a53-856f-73c097dafe7e)

Reply:
ah, was only looking at the sigs ... 4 of them, sigh